test/statistics/bst-statistics-test.ts

```diff
@@ -0,0 +1,23 @@
+import * as assert from "assert";
+import {BstStatistics, BstCommand, BstEvent, SOURCE_API_URL} from "../../lib/statistics/bst-statistics";
+import * as nock from "nock";
+
+describe("BstStatistics", function() {
```

Comment:
add a test for the case were the connection to the endpoint fails, to ensure it fails without throwing errors

Reply:
test added on https://github.com/bespoken/bst/pull/475/commits/464701f9c095d7dfc80c11b036d70ee0c46498d4